src/pages/Pool.vue

```diff
@@ -24,7 +24,16 @@
                 {{ fNum(tokenMeta.weight, 'percent_lg') }}
               </span>
             </div>
-            <LiquidityMiningTooltip :pool="pool" class="-ml-1 mt-2" />
+            <BalChip
+              v-if="pool.dynamic.isNewPool"
+              color="red"
+              size="sm"
+              class="uppercase mt-2 mr-2"
+              :outline="false"
+            >
+              {{ $t('new') }}
+            </BalChip>
+            <LiquidityMiningTooltip :pool="pool" class="-ml-1 mt-1" />
```

Comment:
The liquidity mining looks a bit broken on production so this fixes it (along with the change inside `LiquidityMiningTooltip`)